fastlane/Fastfile

```diff
@@ -467,47 +467,6 @@ platform :ios do
     download_jetpack_localized_app_store_metadata
   end
 
-  #####################################################################################
-  # build_and_upload_beta_release
-  # -----------------------------------------------------------------------------------
-  # This lane builds a beta release of the app and optionally uploads
-  # it for both internal and external distribution
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane build_and_upload_beta_release [skip_confirm:<skip confirm>]
-  #   [create_gh_release:<create release on GH>]
-  #
-  # Example:
-  # bundle exec fastlane build_and_upload_beta_release
-  # bundle exec fastlane build_and_upload_beta_release skip_confirm:true
-  # bundle exec fastlane build_and_upload_beta_release create_gh_release:true
-  #####################################################################################
-  desc 'Builds and uploads a beta release for distribution'
-  lane :build_and_upload_beta_release do |options|
-    build_and_upload_app_center(options)
-    build_and_upload_app_store_connect(options)
-  end
-
-  #####################################################################################
-  # build_and_upload_stable_release
-  # -----------------------------------------------------------------------------------
-  # This lane builds a stable release of the app and optionally uploads
-  # it for distribution
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane build_and_upload_stable_release [skip_confirm:<skip confirm>]
-  #   [create_gh_release:<create release on GH>]
-  #
-  # Example:
-  # bundle exec fastlane build_and_upload_stable_release
-  # bundle exec fastlane build_and_upload_stable_release skip_confirm:true
-  # bundle exec fastlane build_and_upload_stable_release create_gh_release:true
-  #####################################################################################
-  desc 'Builds and uploads a stable release for distribution'
-  lane :build_and_upload_stable_release do |options|
-    build_and_upload_app_store_connect(options)
-  end
-
```

Comment:
These above were just unused lanes from the CircleCI implementation that I noticed while looking around the file.